New package: BraketAHS v0.0.1

JuliaRegistrator opened the pull request:
- Registering package: BraketAHS
- Repository: https://github.com/amazon-braket/BraketAHS.jl
- Created by: @ykharkov
- Version: v0.0.1
- Commit: 35850bc21aee28d32148b8b42468a7f690212f14
- Reviewed by: @ykharkov
- Reference: https://github.com/amazon-braket/BraketAHS.jl/issues/4#issuecomment-1979548875
- Description: Simulating Rydberg atom dynamics with Tensor Networks
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a8359805094e80ec3fa215eddc6fc4a2e3b0faba72dc436749d6c17632ed8fb025f5ecebb36a5094550c8c3477c0e7f3b39a6aee2dff1eec52740086bc94ac11c6725aab34134824f02918f1b38c2ade0e2b25994a91af195ab0be3d9736cce3d2a6de3725f96643ebb0774dff2a71a604735eb783a1c62862eb41de3862645c979d29 -->

asinghvi17 commented:
[noblock]
You need to put compat bounds in your Project.toml, and it looks like you have a relative path here:

https://github.com/amazon-braket/BraketAHS.jl/blob/215cfc58f591d2e4e433d580e5bc21059fdd1609/src/ahs_program.jl#L72

If you must run code on package loading (and not, for example, wrap it in a function) then you need to make this an absolute path, like so:

```julia
file_path = joinpath(dirname(@__DIR__), "examples", "ahs_program.json")
```

since your formulation would not have worked on Windows in any case.

ykharkov commented:
> [noblock] @ykharkov, you need to put compat bounds in your Project.toml, and it looks like you have a relative path here:
> 
> https://github.com/amazon-braket/BraketAHS.jl/blob/215cfc58f591d2e4e433d580e5bc21059fdd1609/src/ahs_program.jl#L72
> 
> If you must run code on package loading (and not, for example, wrap it in a function) then you need to make this an absolute path, like so:
> 
> ```julia
> file_path = joinpath(dirname(@__DIR__), "examples", "ahs_program.json")
> ```
> 
> since your formulation would not have worked on Windows in any case.

[noblock]
Thanks, I fixed it. 

Although now I see the permission denied error: 
`ERROR: LoadError: SystemError: opening file "/tmp/jl_97XlXW/packages/BraketAHS/xa55F/examples/ahs_program.json": Permission denied`.

Can you please advise on how to fix it?

JuliaRegistrator committed:
New package: BraketAHS v0.0.1

UUID: aad1d28c-1a3f-47da-806d-2ed76430c1d6
Repo: https://github.com/amazon-braket/BraketAHS.jl.git
Tree: f0738b70e2e0eadb6b81e7ac04c1ec989d9fd0ae

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

ykharkov commented:
[noblock]
@asinghvi17 The package has been updated, all checks passed. Ready for merge.

asinghvi17 commented:
[noblock] Great!  I am not able to merge here but this should get automerged in about 3 days.
